contents/codes/308.md

```diff
@@ -1,11 +1,13 @@
 ---
 set: 3
 code: 308
-title: Permanent Redirect
+title: Permanent Redirect / Resume Incomplete
```

Comment:
Can we just leave the title as-is? I think `Permanent Redirect` is the predominate use of this code so let's keep it that until `Resume Incomplete` lands in a spec.

Reply:
Then Ill do the same for 103.